app/src/main/res/layout/layout_add_credit_card.xml

```diff
@@ -17,7 +17,7 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginStart="@dimen/credit_cards_saved_cards_item_margin_start"
-        app:srcCompat="@drawable/ic_new"
+        app:srcCompat="@drawable/mozac_ic_new"
```

Comment:
Following this comment https://github.com/mozilla-mobile/fenix/pull/23313/files#r788895326, if you notice  `ic_new.xml ` has a `fillColor` of  `android:fillColor="?primaryText"` but AC version `mozac_ic_new` has a `fillColor` of  `android:fillColor="@color/mozac_ui_icons_fill"` and we are not replacing the fill color here, that will means that we will get `android:fillColor="@color/mozac_ui_icons_fill"`. Is this intentional ?

Reply:
The same could apply for other cases we are replacing with AC drawables